os/binfmt/binfmt_loadbinary.c

```diff
@@ -58,34 +57,31 @@ extern void mpu_user_intsram_context(uint32_t region, uintptr_t base, size_t siz
  *   schedule to unload the module when task exits.
  *
  * Input Parameters:
- *   filename - The path to the program to be executed. If
- *              CONFIG_LIB_ENVPATH is defined in the configuration, then
- *              this may be a relative path from the current working
- *              directory. Otherwise, path must be the absolute path to the
- *              program.
- *
- *   binsize  - The size of ELF binary to be loaded, if this value is less than
- *              will return with invalid parameter.
- *
- *   offset   - The offset from which ELF binary has to be read in MTD partition.
+ *   filename  - The path to the program to be executed. If
+ *               CONFIG_LIB_ENVPATH is defined in the configuration, then
+ *               this may be a relative path from the current working
+ *               directory. Otherwise, path must be the absolute path to the
+ *               program.
  *
- *   ram_size - The size of RAM partition required by this app
+ *   load_attr - This structure contains the information that the binary
```

Comment:
I think binary manager needs the load_attr to load, not binary.

Reply:
I modified it.